test_pull_requests

```diff
@@ -322,13 +322,11 @@ module Hub
     def delete_comment(comment_id, repo, num_tries=3)
       $stderr.puts "  Deleting comment ##{comment_id}"
       (1..num_tries).each do |i|
-        begin
-          res = delete "#{GITHUB_API_BASE_URL}/repos/#{Properties['github_user']}/#{repo}/issues/comments/%s" % [comment_id]
-        rescue Net::HTTPNotFound => e
-          # If we get 404 on a retry, that means there's no comment to delete
-          break if i > 1
-        end
+        res = delete "#{GITHUB_API_BASE_URL}/repos/#{Properties['github_user']}/#{repo}/issues/comments/%s" % [comment_id]
+        # If we get 404 on a retry, that means there's no comment to delete
         break if res.success?
+        # convert to string because it looks like res.code could be int or string depending on ruby
+        break if ("#{res.code}" == "404" && i > 1)
```

Comment:
in my repl it gives me a string. In the other place it's called from in test_pull_requests, it's compared against an int (until this PR)